app/views/dashboard/advisers/_advisers_table.html.erb

```diff
@@ -9,18 +9,29 @@
       </tr>
     </thead>
     <tbody>
-      <% advisers.each do |adviser| %>
-        <tr class="t-advisers-table-row">
-          <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
-          <td class="t-adviser-name"><%= adviser.name %></td>
-          <td>
-            <ul class="l-comma-separated-list">
-              <% adviser.qualifications.each do |qualification| %>
-                <li class="t-adviser-qualification"><%= qualification.name %></li>
-              <% end %>
-            </ul>
+      <% if firm.advisers.present? %>
+        <% firm.advisers.each do |adviser| %>
+          <tr class="t-advisers-table-row">
+            <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
+            <td class="t-adviser-name"><%= adviser.name %></td>
+            <td>
+              <ul class="l-comma-separated-list">
```

Comment:
@benbarnett I can see what you're getting at — do you have a reference for what is and isn't an `l-`? So I can stop trying to model your brain in my head!

We're trashing that code anyway I believe, so no big concern (and it's just a move in this PR).